core/src/actors/peers_manager.rs

```diff
@@ -20,6 +25,37 @@ pub struct PeersManager {
     peers: Peers,
 }
 
+impl PeersManager {
+    /// Method to periodically dump peers into storage
+    fn storage_peers(&self, ctx: &mut Context<Self>, storage_peers_period: Duration) {
```

Comment:
`storage_peers` should be renamed to a more descriptive name: `dump_peers`, `store_peers`, ...